packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -61,6 +61,9 @@ export interface IClusterEngine extends IEngine {
   /** The application used by this engine to perform rotation for a multi-user scenario. */
   readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
 
+  /** The log types that are available with this engine type */
+  readonly logTypes: string[];
```

Comment:
1. Should this be optional? With the understanding that `undefined` means "no validation will happen"?
2. I feel like this name is a little too terse 🙂. `supportedLogTypes` perhaps?

Reply:
> 1. Should this be optional?

My take on this is that we know the defined values for the existing clusters, so there's (currently) no reason to make this optional. Would the motivation just be future-proofing?

> 2. I feel like this name is a little too terse 🙂. supportedLogTypes perhaps?

Sure.